liferay-echo/liferay-portal
# LPS-201366 Migrate portal-reports-engine-console-web to purely javascript

Motivation
=======
The motivation behind this ticket is removal of AUI code and use native functions instead.
https://liferay.atlassian.net/browse/LPS-201366

Steps to Verify
========
1. Go to Configuration -> Reports Admin -> Definition tab -> click on (+) button to add a Definition report
2. Create a template to be able to create a definition : Copy the .jrxml file from https://help.liferay.com/hc/en-us/articles/360018185011-Generating-Reports-in-Liferay#:~:text=a%20file%20named-,sample%2Dreport.jrxml,-somewhere%20on%20your to a local file.
3. In the definition view, in the Reports Parameters section, add different params and check that are being added by saving the report and opening it again.
4. After saving the Definition, go back to its view and click on Add Schedule on the bottom of the view.
5. In the new view search for the Reports Parameter section and check that was working as before.
